packages/labs/gen-manifest/package.json

```diff
@@ -19,7 +19,8 @@
       "dependencies": [
         "../../tests:build",
         "../analyzer:build",
-        "../gen-utils:build"
+        "../gen-utils:build",
+        "../../lit-element:build"
```

Comment:
I don't see that anything in this package imports lit-element... why was this needed?

Reply:
We're building a manifest with types for an example that imports lit, so we need the types for lit